src/features/metadata-instance-editor/messages.js

```diff
@@ -168,6 +168,32 @@ const messages = defineMessages({
         description: 'Label for enable cascade policy toggle switch',
         id: 'boxui.metadataInstanceEditor.enableCascadePolicy',
     },
+    enableAIAutofill: {
+        defaultMessage: 'Box AI Autofill',
+        description: 'Label for enable AI autofill toggle switch',
+        id: 'boxui.metadataInstanceEditor.enableAIAutofill',
+    },
+    aiAutofillDescription: {
+        defaultMessage: 'Use Box AI to automatically extract metadata values.',
+        description: 'Description for AI autofill toggle switch',
+        id: 'boxui.metadataInstanceEditor.aiAutofillDescription',
+    },
+    aiAutofillNotice: {
+        defaultMessage:
+            'Enabling this feature may involve additional charges. Please review our {pricingLink} for more information.',
+        description: 'Notice for AI autofill toggle switch',
+        id: 'boxui.metadataInstanceEditor.aIAutofillNotice',
```

Comment:
i hate myself for pointing this out
```suggestion
        id: 'boxui.metadataInstanceEditor.aiAutofillNotice',
```